HISTORY.md

```diff
@@ -1,8 +1,15 @@
+# TileDB-Py 0.11.3 Release Notes
+
+## Impovements
+* Support for Python 3.10 [#808](https://github.com/TileDB-Inc/TileDB-Py/pull/808)
+
 # TileDB-Py 0.11.2 Release Notes
+
 ## TileDB Embedded updates:
 * TileDB-Py 0.11.2 includes TileDB Embedded [TileDB 2.5.2](https://github.com/TileDB-Inc/TileDB/releases/tag/2.5.2)
 
 # TileDB-Py 0.11.1 Release Notes
+
 ## Bug fixes
 * Support dict parameter for 'config' argument to VFS constructor [#805](https://github.com/TileDB-Inc/TileDB-Py/pull/805)
```

Comment:
Would you mind to move this to the correct section (0.11.2)?

Reply:
Do you mean move "Support dict parameter..." into 0.11.2 or something else?

Reply:
> Do you mean move "Support dict parameter..." into 0.11.2

yes